[2024-04-20T15:28:45.033+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-20T15:28:45.051+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2024-04-20T15:28:45.060+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2024-04-20T15:28:45.061+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-20T15:28:45.074+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): first_task> on 2023-08-23 00:00:00+00:00
[2024-04-20T15:28:45.082+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=674) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-20T15:28:45.082+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2023-08-23T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpq7zh5utu']
[2024-04-20T15:28:45.086+0000] {standard_task_runner.py:63} INFO - Started process 681 to run task
[2024-04-20T15:28:45.086+0000] {standard_task_runner.py:91} INFO - Job 27: Subtask first_task
[2024-04-20T15:28:45.127+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2023-08-23T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-20T15:28:45.173+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-23T00:00:00+00:00'
[2024-04-20T15:28:45.174+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-20T15:28:45.182+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-20T15:28:45.182+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-04-20T15:28:45.186+0000] {subprocess.py:86} INFO - Output:
[2024-04-20T15:28:45.187+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-04-20T15:28:45.187+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-20T15:28:45.188+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-20T15:28:45.199+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, execution_date=20230823T000000, start_date=20240420T152845, end_date=20240420T152845
[2024-04-20T15:28:45.224+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-20T15:28:45.231+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-20T15:28:45.232+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
